includes/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker.php

```
@@ -0,0 +1,241 @@
+			$file_path = $feed->get_file_path();
+			if ( ! empty( $file_path ) && file_exists( $file_path ) ) {
+				wp_delete_file( $file_path );
+			}
+		} catch ( Exception $e ) {
+			WC_Stripe_Logger::error(
+				'Agentic Commerce: Inventory sync failed',
+				[
+					'error' => $e->getMessage(),
+					'code'  => $e->getCode(),
+				]
+			);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'sync_inventory'. (undefined)

(StaticAccess)